suite-native/transactions/src/components/TransactionDetail/TransactionDetailStatusSection.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactNode } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useFormatters } from '@suite-common/formatters';
+import { selectTransactionBlockTimeById, TransactionsRootState } from '@suite-common/wallet-core';
+import { AccountKey, WalletAccountTransaction } from '@suite-common/wallet-types';
+import { isPending } from '@suite-common/wallet-utils';
+import { Icon } from '@trezor/icons';
+
+type TransactionDetailStatusSectionProps = {
+    transaction: WalletAccountTransaction;
+    accountKey: AccountKey;
+};
+
+const summaryColumnStyle = prepareNativeStyle(_ => ({
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    width: 40,
+}));
+
+const statusIconStyle = prepareNativeStyle<{ isTransactionPending: boolean }>(
+    (utils, { isTransactionPending }) => ({
+        backgroundColor: isTransactionPending
+            ? utils.colors.backgroundAlertYellowBold
+            : utils.colors.backgroundPrimaryDefault,
+        borderRadius: utils.borders.radii.round,
+        padding: utils.spacings.small,
+        marginVertical: utils.spacings.small,
+    }),
+);
+const borderLineStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundNeutralSubdued,
+    height: 25,
+    width: 1,
+}));
+
+const VerticalSeparator = () => {
+    const { applyStyle } = useNativeStyles();
+
+    return <Box style={applyStyle(borderLineStyle)} />;
+};
+
+export const SummaryRow = ({
+    children,
+    leftComponent,
+}: {
+    children: ReactNode;
+    leftComponent: ReactNode;
+}) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box flexDirection="row">
+            <Box style={applyStyle(summaryColumnStyle)}>{leftComponent}</Box>
+            <Box marginLeft="medium" justifyContent="center" flex={1}>
+                {children}
+            </Box>
+        </Box>
+    );
+};
+
+export const TransactionDetailStatusSection = ({
+    transaction,
+    accountKey,
+}: TransactionDetailStatusSectionProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { DateTimeFormatter } = useFormatters();
+
+    const transactionBlockTime = useSelector((state: TransactionsRootState) =>
+        selectTransactionBlockTimeById(state, transaction.txid, accountKey),
+    );
+
+    const isTransactionPending = isPending(transaction);
```

Comment:
I would prefer to have selector for this and pass only txid to this component to prevent some rerenders